tests/integration/modules/test_esxi.py

```diff
@@ -457,3 +457,34 @@ def test_esxi_get(service_instance):
         host_name="no_host",
     )
     assert not ret
+
+
+def test_get_ntp_config(service_instance):
+    """
+    Test get ntp configuration on ESXi host
+    """
+    ret = esxi.get_ntp_config(
+        service_instance=service_instance,
+        datacenter_name="Datacenter",
+        cluster_name="Cluster",
+    )
+    assert ret
+    for host in ret:
+        assert sorted(list(ret[host])) == sorted(
+            [
+                "ntp_config_file",
+                "time_zone",
+                "time_zone_description",
+                "time_zone_name",
+                "ntp_servers",
+                "time_zone_gmt_offset",
+            ]
+        )
```

Comment:
**question** can you use the test_value_scraper to grab the expected values, and then assert that the expected == actual? When we shift to actually configuring the ntp values on the hosts for the tests, this test wouldn't really have to be refactored at that point.

Reply:
ok